tokens.json

```diff
@@ -1502,6 +1502,42 @@
             }
           }
         }
+      },
+      "gradients": {
+        "background": {
+          "primary": {
+            "value": "$other.gradients.primary",
+            "type": "color"
+          },
+          "blue": {
+            "value": "{other.gradients.blue}",
+            "type": "color"
+          },
```

Comment:
there is an issue here 2 different type of variable we need to keep with `$other.gradient.primary`